src/Sylius/Bundle/ApiBundle/Resources/config/services/filters.xml

```diff
@@ -132,54 +132,38 @@
             <tag name="api_platform.filter" />
         </service>
 
-<!--        <service id="sylius.api.order_date_filter" parent="api_platform.doctrine.orm.date_filter" public="true">-->
-<!--            <argument type="collection">-->
-<!--                <argument key="checkoutCompletedAt">exclude_null</argument>-->
-<!--            </argument>-->
-<!--            <tag name="api_platform.filter" />-->
-<!--        </service>-->
-
-<!--        <service id="sylius.api.order_product_filter" parent="api_platform.doctrine.orm.search_filter" public="true">-->
-<!--            <argument type="collection">-->
-<!--                <argument key="items.productName">exact</argument>-->
-<!--            </argument>-->
-<!--            <tag name="api_platform.filter" />-->
-<!--        </service>-->
-
-<!--        <service id="sylius.api.order_shipping_method_filter" parent="api_platform.doctrine.orm.search_filter" public="true">-->
-<!--            <argument type="collection">-->
-<!--                <argument key="shipments.method.code">exact</argument>-->
-<!--            </argument>-->
-<!--            <tag name="api_platform.filter" />-->
-<!--        </service>-->
-
-<!--        <service id="sylius.api.order_variants_filter" parent="api_platform.doctrine.orm.search_filter" public="true">-->
-<!--            <argument type="collection">-->
-<!--                <argument key="items.variant.translations.name">exact</argument>-->
-<!--            </argument>-->
-<!--            <tag name="api_platform.filter" />-->
-<!--        </service>-->
+        <service id="sylius_api.admin.search_filter.order" parent="api_platform.doctrine.orm.search_filter" public="true">
```

Comment:
```suggestion
        <service id="sylius_api.search_filter.admin.order" parent="api_platform.doctrine.orm.search_filter" public="true">
```
Up to now, every other section-based service id has the section after the type of service:
- `sylius_api.state_provider.admin.order.adjustment.collection`
- `sylius_api.state_processor.admin.promotion_coupon.remove`
- `sylius_api.doctrine.orm.query_extension.shop.address.shop_user_based`

etc.